packages/fether-electron/src/main/app/constants/index.js

```diff
@@ -0,0 +1,15 @@
+// Copyright 2015-2019 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+// See trustworthiness of 127.0.0.1 vs localhost: https://letsencrypt.org/docs/certificates-for-localhost/
+const TRUSTED_URLS = [
```

Comment:
Note: By adding these, we specifically disallow users using Fether with a remote node. I'm +1ing this decision, but just want to double-check with @Tbaut. ssh tunnels are still possible.

Reply:
Yup that's fine, Fether is for those who want a local (or ssh'ed) node IMO.